pkg/lib/web/javascript.go

```diff
@@ -37,7 +39,7 @@ func (d JavaScriptDescriptor) FindResources(fs resource.Fs) ([]resource.Location
 }
 
 func (d JavaScriptDescriptor) ViewResources(resources []resource.ResourceFile, rawView resource.View) (interface{}, error) {
-	output := bytes.Buffer{}
+	output := readcloserthunk.Reader(strings.NewReader(""))
```

Comment:
Follow my comment on CSS descriptor and compose things using a slice of ReadCloserThunk.